docs/codeReview.md

```diff
@@ -58,14 +59,12 @@ A PR advances from self-review to reviewer-review and finally maintainer review
 - If you need to clarify parts of the code, check if it can be done by adding comments or improve naming of variables/functions/classes
 - When you replied or resolved all comments, move issue back to reviewer- or maintainer-review
 
-## Experiment
+## One Review Required
 
-We want to run an experiment for two months. During the experiment, all PRs will still require a Reviewer Review and a Maintainer Review. However, if the developer creating the PR feels that only one review is required, they can add a label: “One Review Required”. The Reviewer Reviewer then adds a comment saying whether they agree that the PR could be merged once they’ve approved it.
+If the developer creating the PR feels that maintainer review isn't necessary, they can add a label: “One Review Required”. The Reviewer Reviewer then adds a comment saying whether they agree that the PR could be merged once they’ve approved it.
```

Comment:
+1 While I understand the reasoning behind the comment added by the reviewer, in practice I did not see any such comments added. So either we emphasize this more, or we adopt to the lived practice and remove this requirement.

Reply:
Perhaps the Reviewer Reviewer only adds a comment if they disagree with the label as they're more likely to agree than disagree?

Reply:
yes, good point! I'll change that to what @nickoferrall suggested, thank you!

Reply:
@Dschoordsch @nickoferrall, how about `If the Reviewer thinks it's not safe to have just one review, it's his responsiblity to add a comment saying that Maintainer review is required to merge the PR.`?

Reply:
Looks good to me! One small edit - we could say `it's their responsibility` instead

Reply:
oh, that's right! thank you 😅 fixed!